AMQP/rabbitmq/README.md

```diff
@@ -25,6 +26,11 @@ The Gateway container will run a `amqp.proxy` service that allows WebSocket clie
       <!-- Only WebSockets coming from this origin can access this url -->
       <allow-origin>https://kaazing.example.com:8000/</allow-origin>
     </cross-site-constraint>
+
+    <cross-site-constraint>
+      <!-- Only WebSockets coming from this origin can access this url -->
+      <allow-origin>http://kaazing.example.com:8001/</allow-origin>
```

Comment:
I would expect we would need both origins listed, but perhaps this is not the case?  Can you confirm that both wss and ws work